### PR TITLE
Logout confirm page is failing to log the user out on auth-server-wildfly

### LINKED ISSUE #11753: Logout confirm page is failing to log the user out on auth-server-wildfly
### Describe the bug

Cookies tests are failing due to logout url confirm not working correctly

jenkins/job/universal-test-pipeline-server/1444/testReport/junit/org.keycloak.testsuite.broker/KcOidcBrokerEventTest/Run_tests___Cookies_Tests___cookies_tests___loginWithExistingUser/

### Version

18.0.0

### Expected behavior

User confirms logout, goes to user account and is redirected to the login page

### Actual behavior

User confirms logout, goes to user account and the user account is shown as if the user didn't log out

### How to Reproduce?

Cookies tests failing on auth-server-wildfly
- User is logged in
- User goes to logout url
- User click to confirm logout
- A screen is shown with a message that the logout was sucessful
- user goes to the account URL and the user information is shown as it didn't log out

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by mabartos**
Some `Cookies` tests started failing after some changes to the logout process in Keycloak (RP-initiated logout). The main cause of this was the Cookies tests use two separate hostnames and it wasn't possible to rely on the particular cookies. Therefore, it was necessary to define either `id_token_hint` (ID token) or at least `client` (clientId) parameters in order to properly log out the user.

Fixes #11753

### CODE REVIEW

**Comment by mabartos** [comment]
@mposolda @miquelsi Could you please check it?
Pipeline tests are running ATM - https://keycloak-jenkins.com/job/universal-test-pipeline-server/1651/

**Review by mposolda — APPROVED** [review]
@mabartos Thanks! I can merge this PR once @miquelsi approves this as well.

**Review by miquelsi — APPROVED** [review]

**Comment by mabartos** [comment]
@mposolda @miquelsi Thanks!

**Pull request merged**
